### PR TITLE
Fixed card in myGearForRent

### PR DESCRIPTION
**Pull request by Tien9320**
Screen Shot 2021-08-20 at 2 56 08 PM

### CODE REVIEW

File: app/views/listings/my_listings.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
               <div class="card-product-content">
                 <h2><%= number_to_currency(listing.price) %><small>/day</small></h2>
                 <h2 class="mt-2"><%= listing.name %></h2>
-                <p><%= listing.description %></p><br>
+                <!-- <p><%= listing.description %></p><br> -->
```
Comment (MmeIsabelle) [inline app/views/listings/my_listings.html.erb:16]: I think you can remove it

**Review by MmeIsabelle — CHANGES_REQUESTED** [review]
You could completely remove the commented line 16

**Comment by Tien9320** [comment]
I'll do that. I put that just in case one of us still want the desc

**Comment by MmeIsabelle** [comment]
Ready for a merge

**Pull request merged**